### PR TITLE
[ADD] 8.0 stock_picking_send_by_mail

### PR DESCRIPTION
**Pull request by sandrafig**
Adds a module that allows to send the stock picking report by email

### CODE REVIEW

File: doc/cla/individual/sandrafig.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Spain, 2015-12-22
```
Comment (rousseldenis) [inline doc/cla/individual/sandrafig.md:1]: This is not needed. Follow https://odoo-community.org/page/cla

File: stock_picking_reorder_lines/stock.py [hunk 2]
```diff
@@ -46,9 +46,12 @@ class StockPicking(models.Model):
     @api.depends('move_lines')
     def _get_max_line_sequence(self):
         for picking in self:
-            picking.max_line_sequence = (
-                max(picking.mapped('move_lines.sequence')) + 10
-                )
+            if picking.move_lines:
```
Comment (rousseldenis) [inline stock_picking_reorder_lines/stock.py:49]: This file is not relevant here. Remove these changes

File: stock_picking_send_by_mail/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Sandra Figueroa Varela>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock Picking by Mail",
+    "summary": "Send stock picking by email",
+    "version": "8.0.1.0.0",
+    "author": "<Sandra Figueroa Varela>, Odoo Community Association (OCA)",
+    "website": "",
```
Comment (rousseldenis) [inline stock_picking_send_by_mail/__openerp__.py:10]: Please modify this entry following https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py template

File: stock_picking_send_by_mail/models/stock_picking.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Sandra Figueroa Varela>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.osv import osv
+
+
+class StockPicking(osv.osv):
```
Comment (rousseldenis) [inline stock_picking_send_by_mail/models/stock_picking.py:8]: You can inspire you by https://github.com/OCA/stock-logistics-workflow/pull/353/files#diff-602dd04aa39038b2474d280f13cc567f

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some comments. @sandrafig Is this relevant yet ?

File: stock_picking_send_by_mail/models/stock_picking.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# © <2015> <Sandra Figueroa Varela>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.osv import osv
```
Comment (cubells) [inline stock_picking_send_by_mail/models/stock_picking.py:5]: Old api, please change.

**Review by cubells — CHANGES_REQUESTED** [review]

**Comment by quentinDupont** [comment]
Hello @sandrafig.
I'm interested in this module and would like to know if you plan to do the requested changes ?
Thanks !

**Comment by sandrafig** [comment]
Hi @quentinDupont 
Right now I don't have time to make the requested changes

**Comment by pedrobaeza** [comment]
Module is already finished for v9: https://github.com/OCA/stock-logistics-workflow/tree/9.0/stock_picking_send_by_mail, so backporting it would be easy. This one is almost finished in fact.

**Comment by quentinDupont** [comment]
I never backport a module and I don't find a guide or tutorial to backport properly a module.
Where can I find a good ressource ? :)

**Comment by rousseldenis** [comment]
> I never backport a module and I don't find a guide or tutorial to backport properly a module.
> Where can I find a good ressource ? :)

You can follow this guide which is for forwardport. Just inverse origin and destination branch in git format patch command. https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0

**Comment by quentinDupont** [comment]
> > I never backport a module and I don't find a guide or tutorial to backport properly a module.
> > Where can I find a good ressource ? :)
> 
> You can follow this guide which is for forwardport. Just inverse origin and destination branch in git format patch command. https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0

Thanks i'll try this later ! :)

**Comment by quentinDupont** [comment]
I open an other PR with commit history : https://github.com/OCA/stock-logistics-workflow/pull/492
Hope this is good

**Pull request closed without merging**